app/controllers/candidate_interface/after_sign_in_controller.rb

```diff
@@ -14,7 +14,11 @@ def interstitial
         flash[:warning] = "You have already selected #{course.name_and_code}."
         redirect_to candidate_interface_course_choices_review_path
       elsif current_application.has_the_maximum_number_of_course_choices?
-        flash[:warning] = I18n.t('errors.messages.too_many_course_choices', course_name_and_code: course.name_and_code)
+        if current_application.apply_1?
+          flash[:warning] = I18n.t('errors.messages.too_many_course_choices', course_name_and_code: course.name_and_code)
+        else
+          flash[:warning] = I18n.t('errors.messages.apply_again_course_already_chosen', course_name_and_code: course.name_and_code)
+        end
```

Comment:
Could we do something like:
```suggestion
        error_message_key = current_application.apply_1? ? 'errors.messages.too_many_course_choices' : 'errors.messages.apply_again_course_already_chosen'
        flash[:warning] = I18n.t(error_message_key, course_name_and_code: course.name_and_code)
```

Reply:
have updated 👍 Thanks for the review